aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -195,10 +195,10 @@ module aptos_framework::code {
 
     /// Checks whether the given package is upgradable, and returns true if a compatibility check is needed.
     fun check_upgradability(
-            old_pack: &PackageMetadata, new_pack: &PackageMetadata, new_modules: &vector<String>) {
+        old_pack: &PackageMetadata, new_pack: &PackageMetadata, new_modules: &vector<String>) {
```

Comment:
This is not a good formatting rule. The parameters should not be intended the same as the body